chart/templates/worker-deployment.yaml

```diff
@@ -38,9 +38,33 @@ spec:
       securityContext:
         fsGroup: {{ $root.Values.fsGroup }}
       volumes:
+        - name: vets-api-mockdata
+          emptyDir: { }
+        - name: clamav-volume
+          emptyDir: { }
+        - name: ddsocket
+          hostPath:
+            path: /var/run/datadog
 {{ toYaml .Values.common.volumes | indent 8 }}
       {{- if .Values.mockData.enabled }}
       initContainers:
+        - name: vets-api-mockdata
+          image: 008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/git:latest
+          args:
+            - clone
+            - --single-branch
+            - --
+            - https://$(GITHUB_TOKEN)@github.com/department-of-veterans-affairs/vets-api-mockdata.git
```

Comment:
Not sure `$(GITHUB_TOKEN) `will be loaded in here since this is technically hardcoded.

Reply:
In this case, GITHUB_TOKEN is an environment variable coming from a secret - the secret is still being created in the ENV manifest helm charts, and so the value will be available in this context